train/scripts/dataset.py

```diff
@@ -48,13 +50,19 @@ def __init__(
             atmos_filepath = Path(atmos_filepath)
 
         self.static_vars_ds = xr.open_dataset(
-            data_path / static_filepath, engine="netcdf4"
+            data_path / static_filepath,
+            engine="netcdf4",
+            chunks={} if use_dask else None,
         )
         self.surf_vars_ds = xr.open_dataset(
-            data_path / surface_filepath, engine="netcdf4"
+            data_path / static_filepath,
```

Comment:
```suggestion
            data_path / surface_filepath,
```
I'm not sure if this is intentional or not, but without loading `surface_filepath` here I get a key error (`"t2m"` missing on line 68).

Reply:
Oh yeah. Thanks. Fixed.